src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private final DriverService driverService;
+    private final CarService carService;
+
+    {
+        Injector injector = Injector.getInstance("mate");
+        carService = (CarService) injector.getInstance(CarService.class);
+        driverService = (DriverService) injector.getInstance(DriverService.class);
+    }
+
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("mate");
    private final DriverService driverService = 
                          (DriverService) injector.getInstance(DriverService.class);
    private final CarService carService = 
                          (CarService) injector.getInstance(CarService.class);
```
make those changes in each controller accordingly 
![image](https://user-images.githubusercontent.com/81781479/184757076-126a4226-df49-406c-a960-fef9e2c227bc.png)

Reply:
done